src/main/java/liuni/services/TwitterService.java

```diff
@@ -24,12 +22,16 @@
     private final static Logger logger = LoggerFactory.getLogger(TwitterService.class);
     private static TwitterService INSTANCE = null;
 
-    private TwitterAccountConfig twitterAccountConfig;
-    private Twitter twitter;
+
+    public Twitter twitter;
+
+    @Inject
+    public TwitterService(Twitter twitter) {
+        this.twitter = twitter;
+        getInstance();
```

Comment:
what is this doing? i don't believe you need this

Reply:
Dagger cannot inject on a private constructor, and from my understanding Singletons needed a private constructor and the getInstance() method, so this was my attempt at working around that. Should I try a different technique?